include/dsVideoPort.h

```diff
@@ -188,7 +188,8 @@ dsError_t  dsIsVideoPortEnabled(intptr_t handle, bool *enabled);
 /**
  * @brief Checks whether the specific video port is connected to display.
  * 
- * This function is used to check whether video port is connected to a display or not.
+ * For source devices, this function is used to check whether video port is connected to a display or not.
+ * For sink devices, this function returns display connection status true and dsERR_NONE always.
```

Comment:
Should we give reference to dsVIDEOPORT_TYPE_INTERNAL and ask to return TRUE for Sink device for this handle?

Reply:
Updated